tests/test_nltk_rest.py

```diff
@@ -136,5 +136,37 @@ def test_words_snowball_stemmer_no_auth_content(self):
         self.assertEqual(b'<h1>Authentication Required!</h1>', response.data)
 
 
+class RequestWordsLemmatizerTestCase(NltkRestTestCase):
```

Comment:
Can we update this classe name to RequestLemmatizerTestCase

Reply:
done